src/IntegratedExperts/BehatScreenshot/ScreenshotContext.php

```diff
@@ -89,6 +97,25 @@ public function __construct($parameters = [])
     }
 
     /**
+     * Init function before tests run.
+     *
+     * @param BeforeSuiteScope $scope
+     *
+     * @BeforeSuite
+     */
+    public static function setup(BeforeSuiteScope $scope)
+    {
+        foreach ($scope->getSuite()->getSetting('contexts') as $context) {
+            self::$purge = (isset($context['IntegratedExperts\BehatScreenshot\ScreenshotContext'][0]['purge'])) ? $context['IntegratedExperts\BehatScreenshot\ScreenshotContext'][0]['purge'] : self::$purge;
```

Comment:
this logic is spread too much. It should check env var and if it was set, use the value from there, otherwise, use the value from settings.

also, `$context['IntegratedExperts\BehatScreenshot\ScreenshotContext'][0]` should be assigned to a variable to not dragging it through code.